docs/src/main/asciidoc/spring-integration-support.adoc

```diff
@@ -306,12 +299,47 @@ class Demo{
 }
 ----
 
+===== Configure EventHubsMessageConverter to customize ObjectMapper
+`EventHubsMessageConverter` is made as a configurable bean to allow users to customize ObjectMapper.
+
+[#si-eh-batch]
+===== Batch Consumer Support
+To consume messages from Event Hubs in batches is similar with the above sample, besides users should set the batch-consuming related configuration options for `EventHubsInboundChannelAdapter`.
+
+When create `EventHubsInboundChannelAdapter`, the listener mode should be set as `BATCH`. And set the checkpoint mode as either `MANUAL`
+or `BATCH` as needed.
+
+[source,java]
+----
+@Bean
+class Demo{
+    private static final String INPUT_CHANNEL = "input";
+    private static final String EVENTHUB_NAME = "eh1";
+    private static final String CONSUMER_GROUP = "$Default";
+
+    public EventHubsInboundChannelAdapter messageChannelAdapter(
+            @Qualifier(INPUT_CHANNEL) MessageChannel inputChannel,
+            EventHubsProcessorContainer processorContainer) {
+        //Supported checkpoint modes are MANUAL AND BATCH only.
+        CheckpointConfig config = new CheckpointConfig(CheckpointMode.MANUAL);
+
+        EventHubsInboundChannelAdapter adapter =
+                new EventHubsInboundChannelAdapter(processorContainer, EVENTHUB_NAME,
+                        CONSUMER_GROUP, ListenerMode.BATCH, config);
+        adapter.setOutputChannel(inputChannel);
+        return adapter;
```

Comment:
the config is no longer a parameter of the EventHubsInboundChannelAdapter's ctor